<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandingPage</class>
 <widget class="QMainWindow" name="LandingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Vaccination Data System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_hcw">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Health-Care Worker</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ptntReg">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient Registration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ptntUpdt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Appointment</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
